src/extension.ts

```diff
@@ -41,10 +41,23 @@ export function startServer(context: ExtensionContext) {
     const module = context.asAbsolutePath(
         path.join('out', 'server', 'server.js'),
     );
+
+    var java = '';
+    const config = workspace.getConfiguration('viperSettings');
+    if (config.javaSettings.javaBinary) {
```

Comment:
is this allowed if `config` is `null`? I.e. accessing of a field in `null` gives `null`?

Reply:
I doubt it :-)

Reply:
TypeScript has a "null soaking" syntax for this situation: `config?.javaSettings?.javaBinary`